test/code/plugins/oms_common_test.rb

```diff
@@ -172,11 +172,17 @@ def test_get_os_version_missing_field()
 
     def test_get_hostname
       $log = MockLog.new
+      file_hostname = '/etc/hostname'
```

Comment:
you need to have at least 2 test cases: with the /etc/hostname, and w/o /etc/hostname.